core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -1404,7 +1404,7 @@ public void requestOffhandSwap() {
 
     @Override
     public String name() {
-        return null;
+        return playerEntity.getUsername();
```

Comment:
player entity can be null when this is called - I've made a PR to attempt to deal with this, but essentially, it should imo return the bedrock username if we don't have the username yet (aka, when the player entity is null)

Reply:
resolved in a separate PR: https://github.com/GeyserMC/Geyser/commit/e697eb3ae38bbda0b1b4803ea03a73067881dec2